plugins/auth0/pkg/fetch/fetch.go

```diff
@@ -239,17 +252,13 @@ func (f *Fetcher) getRoles(ctx context.Context, opts []management.RequestOption)
 	return roles.Roles, roles.HasNext(), nil
 }
 
-func (f *Fetcher) getUserRoles(ctx context.Context, uID string) ([]map[string]interface{}, error) {
+func (f *Fetcher) getUserRoles(ctx context.Context, uID string) ([]map[string]any, error) {
```

Comment:
We're being inconsistent in our naming convention. Why do we `fetchGroups` but `getUserRoles`? This function is defined on a `Fetcher` so I'd go with `fetchUserRoles`, `fetchOrgs`, etc.